lib/lightspeed/sale_line.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'resource'
+require_relative 'item'
+
+module Lightspeed
+  class SaleLine < Lightspeed::Resource
+    alias_method :archive, :destroy
```

Comment:
As these don't appear to be archiveable, I'd rather there was no archive method on these, and keep the archive method only on those things that can be archived.